test/index.js

```diff
@@ -1,15 +1,14 @@
 'use strict';
 
-import test from 'tape';
+import test from 'blue-tape';
 import hello from '../source/index';
 
 
-test('Tests run', (assert) => {
-  assert.pass('Tests run');
-  assert.end();
-});
+test('Tests run', (assert) => Promise.resolve(
```

Comment:
Or you could do this... if the function were actually async, this is probably closer to what you'd want to write:

```js
test('Tests run', (assert) => new Promise((resolve) => {
  assert.pass('Tests run');

  resolve();
}));
```